src/constructors/regular.c

```diff
@@ -139,6 +139,155 @@ igraph_error_t igraph_star(igraph_t *graph, igraph_integer_t n, igraph_star_mode
     return IGRAPH_SUCCESS;
 }
 
+/**
+ * \ingroup generators
+ * \function igraph_wheel
+ * \brief Creates a \em wheel graph, a union of a star graph and a
+ *        cycle graph. Note that in case of a graph with 2 vertices,
+ *        the non-center vertex will have a self loop (may imagine
+ *        the earth is the center of the wheel, the moon and its orbit
+ *        is the rim of the wheel).
```

Comment:
The `\brief` line should contain a very brief description only. See the documentation guidelines here: https://github.com/igraph/igraph/wiki/Documentation-guidelines

The rest should go into a separate paragraph.

Reply:
Rewrite and make it a separate paragraph.